guides/hack/02-source-code-fundamentals/37-namespaces.md

```diff
@@ -52,11 +58,16 @@ namespace NS3\Sub1;
 ...                   // __NAMESPACE__ is "NS3\Sub1"
 }
 ```
+## Importing from other Namespaces
+With the `use` keyword, a namespace can import one or more member names into a scope, optionally giving them each an alias.
 
-In this case, the namespace extends from the opening brace to the closing brace.
+When importing many names, use `{ ... }`.
+
+```Hack
+use NS1\{C, I, T}; // instead of `NS1\C, NS1\I, NS1\T`
+```
 
-A namespace can import&mdash;that is, get access to&mdash;one or more names into a scope, optionally giving them each an alias.
-Each of those names may designate a namespace, a sub-namespace, a class, an interface, or a trait.  For example:
+Imported names can designate a namespace, a sub-namespace, a class, an interface, or a trait, as shown in the following example:
```

Comment:
- also functions
- any type, not just classes, interfaces, and traits. e.g. type alises to shapes are OK too.